tests/flakyfunctional/cylc-poll/16-execution-time-limit.t

```diff
@@ -33,6 +33,7 @@ run_ok "${TEST_NAME_BASE}-validate" cylc validate "${WORKFLOW_NAME}"
 workflow_run_ok "${TEST_NAME_BASE}-run" \
     cylc play --reference-test -v --no-detach "${WORKFLOW_NAME}" --timestamp
 #-------------------------------------------------------------------------------
+# shellcheck disable=SC2317
```

Comment:
SC2317 is a warning about unreachable code. In this case as we are doing `run_ok $TEST_NAME cmp_times ...` shellcheck is unable to determine whether we are calling the function because we are passing it by name as an argument to a different function.